src/controllers/accomodation.controller.js

```diff
@@ -0,0 +1,62 @@
+import response from '../helpers/response.helper';
+import accomodation from '../services/accomodation.services';
+import UserServices from '../services/user.service';
+
+/**
+* Class for Travel Admins dealing with Accomodation
+*/
+class AccomdationController {
+  /**
+   * This function helps to to filter and store accomodation information
+   * @param { String } name name of accomodation
+   * @param { String } description description of accomodation
+   * @param { Integer } locationId location of accomodation
+   * @param { String } category category of accomodation
+   * @param { String } owner owner of accomodation
+   * @param { Array } images images of accomodation
+   * @param { Array } rooms rooms of accomodation
+   * @returns {Object} return created Accomodation
+   */
+  static async saveAccomodationInfo(name, description, locationId, category, owner, images, rooms) {
+    const newAccomodation = {
+      name, description, locationId, category, owner
+    };
+    const createdaccomodation = await accomodation.createAccomodation(newAccomodation);
+    const accomodationId = createdaccomodation.dataValues.id;
+    images.forEach(async ({ imageUrl }) => {
+      const newImage = { recordId: accomodationId, imageType: 'accomodation', imageUrl };
+      await accomodation.uploadImage(newImage);
```

Comment:
We need an image service to only manage (CRUD operations) images so that any controller can use it.